<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphAlgorithmViewWidget</class>
 <widget class="QWidget" name="GraphAlgorithmViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Utility</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/window_icons/windowicons/graph_icon.png</normaloff>:/window_icons/windowicons/graph_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Traversal Algorithm:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algo_comboBox">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Depth-First Traversal (DFS)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Breadth-First Traversal (BFS)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minimum Spanning Tree (MST)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dijkstra's Algorithm (Shortest Path)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showGraph_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show Graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="select_city">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="starting_Label">
        <property name="text">
         <string>Starting City:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="from_Label">
        <property name="text">
         <string>From :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="city1_comboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showGraph2_pushButton">
        <property name="text">
         <string>Show Graph</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Viewing: Nothing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="output_plainTextEdit">
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeWindow_pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
